File: CheckIsogram.py
# Check Isogram
# - Determine if a word `s` has no repeating letters.
# - Constraints:
#   - 1 <= len(s) <= 1000

class Solution:
    def isIsogram(self, s: str) -> bool:
        s = s.lower()
        seen = set()
        for c in s:
            if c.isalpha():
                if c in seen:
                    return False
                seen.add(c)
        return True

File: CheckIsogram.cs
// Check Isogram
// - Determine if a word `s` has no repeating letters.

using System;
using System.Collections.Generic;

class Solution
{
    public bool IsIsogram(string s)
    {
        s = s.ToLower();
        HashSet<char> seen = new HashSet<char>();

        foreach (char c in s)
        {
            if (Char.IsLetter(c))
            {
                if (seen.Contains(c))
                    return false;
                seen.Add(c);
            }
        }

        return true;
    }
}
